eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala

```diff
@@ -14,9 +14,22 @@ import com.evolutiongaming.skafka.Topic
 import com.evolutiongaming.smetrics.MeasureDuration
 import com.evolutiongaming.sstream.Stream
 
-
+/** Creates read-only representation of the data stored to Cassandra.
+  *
+  * It is intended to be used for journal recovery. One cannot use Kafka only to
+  * recover because Kafka may have the retention window expried for the older
+  * journals or the offset might be too far in the past for such recovery to be
+  * practical.
+  *
```

Comment:
* or during recovery from Kafka only you would need to filter out too much unrelevant data